swe_roe/ex2b.c

```diff
@@ -605,6 +623,21 @@ typedef struct RDyMesh {
   PetscInt *nG2A;
 } RDyMesh;
 
+/// @brief Computes the cross product of two 3D vectors
+/// @param a A RDyVector a
+/// @param b A RDyVector b
+/// @param c A RDyVector c
+/// @return 0 on success, or a non-zero error code on failure
+PetscErrorCode VecCrossProduct(RDyVector a, RDyVector b, RDyVector *c) {
```

Comment:
Not a big deal, but consider `CrossProduct` instead, especially if we put this into the `RDycore` repo. The `Vec` prefix could be confusing if people think it relates to the PETSc `Vec` type.